lammps/bin/run_lammps_2024.02.07_mpi.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+mpiexec -n ${PYIRON_CORES:=$1} --oversubscribe lmp_mpi -in control.inp;
```

Comment:
Is this intending to get the conda version of lammps from the `pyiron/...` module?  Then I would like to make that more explicity by

```suggestion
module purge
module pyiron
mpiexec -n ${PYIRON_CORES:=$1} --oversubscribe lmp_mpi -in control.inp;
```

and renaming the script to include conda or so "conda", since in the current way it's not actually guaranteed that we end up with lammps 2024 Feb07.

Reply:
I guess we have some confusion here - these are the general pyiron resources people download when they install pyiron from conda, so yes it is using LAMMPS from conda and by default there is no module system available.